edison/static/js/policy.controller.js

```diff
@@ -5,6 +5,61 @@ import { PolicyElementsModifier } from './policyElementsModifier.js'
 var policy = new Policy();
 var countCondition = 0;
 
+var policiesExamples = [];
+var policyTest1 = new Policy('Test Policy One', 'living room', 'light On, shutters Off', 'humidity < 40, tempetrue < 25');
+var policyTest2 = new Policy('Test Policy Two', 'kichen', 'shutters Off', 'humidity > 80, tempetrue between 30 50');
+policiesExamples.push(policyTest1);
+policiesExamples.push(policyTest2);
+
+function setCardBody(policy) {
+    var cardBody = createInitElement('div', 'card-body');
+    var cardTitle = createInitElement('h5', 'card_title');
+    var cardText = createInitElement('h5');
+
+    cardBody.appendChild(cardTitle);
+    cardBody.appendChild(cardText);
+    cardText.innerHTML = `this policy is setting to the ${policy.room} and the policy do ${policy.command}
+    with these conditions: ${policy.condition}`;
+
+    return cardBody;
+}
+
+function createCard(policyNum, policy) {
+    var card = createInitElement('div', 'card');
+    var headerCard = createInitElement('div', 'card-header');
+    var buttonTextSize = createInitElement('h5', 'mb-0');
+    var headerCardButton = createInitElement('button', 'btn btn-link');
+
+    card.appendChild(headerCard);
+    headerCard.appendChild(buttonTextSize);
+    headerCardButton.setAttribute("data-toggle", "collapse");
+    headerCardButton.setAttribute("data-target", "#collapse-policy" + policyNum);
+    headerCardButton.setAttribute("aria-expanded", "true");
+    headerCardButton.setAttribute("aria-controls", "collapse-policy" + policyNum);
+    headerCardButton.innerHTML = policy.name;
+    buttonTextSize.appendChild(headerCardButton);
+
+    return card;
+}
+
+function showPolicies() {
+    //once the route will be ready this function will be modifiy to get the policies from the backend and convert them to array 
+    var len = policiesExamples.length;
+    var showPolicies = document.getElementById('show-policies');
+
+    for (var i = 0; i < len; i++) {
+        var card = createCard(i, policiesExamples[i]);
```

Comment:
same as the comment about the "policiesExample"

Reply:
Ok

Reply:
look at the more detailed comment below